qemu/tests/cfg/hv_avic.cfg

```diff
@@ -5,5 +5,7 @@
     required_qemu = [6.2.0,)
     cpu_model_flags += ",hv_avic"
     cpuid_chk_cmd = "rpm -qa | grep cpuid"
-    cpuid_pkg = ""
+    cpuid_pkg = "cpuid"
+    RHEL.9:
+        repo_install_cmd = "yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm"
```

Comment:
The limited QEMU version - QEMU 6.2 is used by RHEL8.6.  
Setting the epel repo for RHEL.9 is not enough,   and it is not friendly for the new product version.

Reply:
Assume the fedora epel repository naming rule is consistent:
`epel-release-latest-$RHEL_X_VERSION.noarch.rpm`.

Suggest replace the RHEL_X_VERSION during case running.